pkg/status/combinedstatus-resolution.go

```diff
@@ -657,7 +705,7 @@ func calculateCombinedFieldAggregation(combinedFieldNamedAgg v1alpha1.NamedAggre
 
 			sum += *subject
 		}
-		numStr = fmt.Sprintf("%f", sum)
+		numStr = strconv.FormatFloat(sum, 'f', -1, 64)
```

Comment:
Wouldn't `'g'` be the more natural value for `fmt` here?

Reply:
I saw that it made no difference on the examples I tried, done anyway.